nullaway/src/main/java/com/uber/nullaway/fixserialization/out/SuggestedNullableFixInfo.java

```diff
@@ -26,64 +26,56 @@
 import com.sun.source.util.TreePath;
 import com.uber.nullaway.ErrorMessage;
 import com.uber.nullaway.fixserialization.location.FixLocation;
-import com.uber.nullaway.fixserialization.qual.AnnotationConfig;
 import java.util.Objects;
 
-/** Stores information suggesting a type change of an element in source code. */
-public class SuggestedFixInfo {
+/** Stores information suggesting adding @Nullable on an element in source code. */
+public class SuggestedNullableFixInfo {
 
   /** FixLocation of the target element in source code. */
   private final FixLocation fixLocation;
   /** Error which will be resolved by this type change. */
   private final ErrorMessage errorMessage;
   /** Suggested annotation. */
```

Comment:
Remove this comment as well.

Reply:
8ad5337fec2971fbb39796e6e370396ddc07ef28